core/classes/Core/URL.php

```diff
@@ -85,6 +85,16 @@ private static function buildNonFriendly(string $url, string $params): string {
         return (defined('CONFIG_PATH') ? CONFIG_PATH : '') . '/index.php?route=' . $url . ((substr($url, -1) == '/') ? '' : '/');
     }
 
+    /**
+     * Build an asset path
+     *
+     * @param string $path Contains the asset path relative to the root Nameless directory
+     * @return string
+     */
+    public static function buildAssetPath(string $path): string {
```

Comment:
Reminder to self to do a bulk replacement and utilize this everywhere! Great addition